collins/collins.go

```diff
@@ -155,15 +167,8 @@ func NewClientFromYaml() (*Client, error) {
 	return NewClient(creds.Username, creds.Password, creds.Host)
 }
 
-func openYamlFile() (io.Reader, error) {
-	yamlPaths := []string{
-		os.Getenv("COLLINS_CLIENT_CONFIG"),
-		path.Join(os.Getenv("HOME"), ".collins.yml"),
-		"/etc/collins.yml",
-		"/var/db/collins.yml",
-	}
-
-	for _, path := range yamlPaths {
+func openYamlFile(paths []string) (io.Reader, error) {
```

Comment:
you mifght want to name this openYamlFiles() since it takes an array of file paths and not a single one.  YMMV